client-src/elements/form-field-specs.js

```diff
@@ -353,6 +354,10 @@ export const ALL_FIELDS = {
     help_text: html`
         Link to the first public proposal to create this feature, e.g.,
         a WICG discourse post.`,
+    extra_help: html`
+        Add a reference to your <a href="https://github.com/WICG/proposals#what-does-a-proposal-look-like">
+        WICG proposal</a>
+        and start socializing the problem with other vendors and developers.`
```

Comment:
Since WICG doesn't use discourse anymore, how about:

```suggestion
        Link to the first public proposal to create this feature.`,
    extra_help: html`
        If there isn't another obvious place to propose your feature, create a
        <a href="https://github.com/WICG/proposals#what-does-a-proposal-look-like">WICG proposal</a>.
        You can use your proposal document to help you socialize the problem with other vendors
        and developers.`
```

I'm not certain we need that last sentence; it's not necessarily connected to having the public proposal document.

Reply:
done.